Resources/ServerInfo/Guidebook/Antagonist/Traitors.xml

```diff
@@ -35,11 +35,13 @@
 
   ## List of Possible Tasks
 
-  - Kill a randomly selected non-traitor.
-  - Kill a randomly selected department head.
+  - Kill or maroon a randomly selected non-traitor.
+  - Kill or maroon a randomly selected traitor.
+  - Kill or maroon a randomly selected department head.
   - Keep a randomly selected traitor alive.
   - Escape on the evacuation shuttle alive and uncuffed.
   - Help a randomly selected traitor finish 2/3 of their objectives.
+  - Die a glorious death.
   - Steal the Captain's [color=#a4885c]ID Card[/color].
   <Box>
```

Comment:
Do we not have a code backing to do this?

Reply:
unfortunately no though it'd be nice.